package.json

```diff
@@ -35,17 +35,18 @@
   "devDependencies": {
     "@types/benchmark": "^1.0.31",
     "@types/jest": "^26.0.13",
+    "@types/lru-cache": "^5.1.1",
     "@types/node": "^12.0.0",
     "@types/uuid": "^8.3.4",
     "benchmark": "^2.1.4",
     "jest": "^26.4.2",
-    "node-abi": "^2.21.0",
     "prebuild": "^11.0.2",
     "prettier": "^2.0.5",
     "ts-node": "^9.0.0",
     "typescript": "^3.9.0"
   },
   "dependencies": {
+    "lru-cache": "^6.0.0",
```

Comment:
Very minor suggestion, happy to skip, but we could use `quick-lru` here to not introduce another lru cache library into Desktop. QuickLRU could end up rooting up to 2x the notifications objects we set as `max` but that shouldn't be a problem.

Reply:
Sure!! I saw `lru-cache` in our yarn.lock so I thought I was actually saving one dependency here 😂  But yeah, I'll switch to `quick-lru` since we're using it in our code 👍 

Done in 2ad4e30a881381e602f6c17e1f4924903f45c676!